snippets/sdp/gpio/icmsg/flpr.conf

```diff
@@ -0,0 +1,2 @@
+CONFIG_SDP_GPIO_BACKEND_ICMSG=y
+CONFIG_IPC_SERVICE=y
```

Comment:
```suggestion
```
If you add it to application Kconfig then you can delete it from here?

Reply:
Done